src/frontends/paddle/tests/test_models/gen_scripts/generate_2in_2out_dynbatch.py

```diff
@@ -63,5 +63,12 @@
 var = [relu3a, relu3b]
 res_paddle = exe.run(fluid.default_main_program(), fetch_list=var, feed=inp_dict)
 
-save_inference_model(os.path.join(sys.argv[1], "2in_2out_dynbatch"), list(inp_dict.keys()), var, exe,
-                              model_filename="2in_2out_dynbatch.pdmodel", params_filename="2in_2out_dynbatch.pdiparams")
+
+# save_inference_model(os.path.join(sys.argv[1], "2in_2out_dynbatch"), list(inp_dict.keys()), var, exe,
```

Comment:
remove the comment

Reply:
sorry I'm still debugging. There are some errors caused.

Reply:
> sorry I'm still debugging. There are some errors caused.

@meiyang-intel  I meet a new problem. In some test causes there are no variables to save. In old api it will still save a .pdiparam file.

```
paddle/static/io.py:610: UserWarning: no variable in your model, please ensure there are any variables in your model to save
```

This will cause a load error:

<img width="959" alt="image" src="https://github.com/openvinotoolkit/openvino/assets/55493212/eb0394e9-b0ef-4676-95c8-d0a23b992bd9">

Reply:
Yes, openvino paddle frontend doesn't support the case of no pdiparam. Can you support it in openivno paddle frontend?

Reply:
> Yes, openvino paddle frontend doesn't support the case of no pdiparam. Can you support it in openivno paddle frontend?

I'd like to help with it. But I do not have any implementation ideas at present, can you provide me with some implementation ideas please?

Reply:
You can support it in this function. https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/paddle/src/frontend.cpp#L419

Reply:
Don't forget remove the debug code after it's fixed.